### ISSUE TITLE
VsCode don't display the add-on 

### ISSUE THREAD

**Issue by mambari** [PsykoSoldi3r/vscode-git-flow#4]
Hi again @PsykoSoldi3r ^^
I have an issue on Mac with vscode.
I have a new project using git-flow

     (JiraDashboard) ∞ 11:25 mehdiambari ➜  JiraDashboard git:(develop) git flow feature start jiraconnect
     Switched to a new branch 'feature/jiraconnect'
     
     Summary of actions:
     - A new branch 'feature/jiraconnect' was created, based on 'develop'
     - You are now on branch 'feature/jiraconnect'
     
     Now, start committing on your feature. When done, use:
     
          git flow feature finish jiraconnect

But when I am on VSC insider (last version)
![image](https://user-images.githubusercontent.com/10997358/99182730-0bd31700-2737-11eb-89a7-2769a8ff013d.png)

### My version of VSC
```
Version: 1.52.0-insider
Commit: ee46c1f89ebb213e0827886f1688c0fbb55be49f
Date: 2020-11-13T06:04:37.031Z (2 days ago)
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
```

**Comment by PsykoSoldi3r** [comment]
Do you have this issue also after restarting vscode? The check of git-flow only happens on launch of a workspace.

**Comment by mambari** [comment]
Yes I do even after restarting vscode.

**Comment by strauss69400** [comment]
I'm the same problem. did you find a solution ?

thanks.

**Comment by PsykoSoldi3r** [comment]
@mambari @strauss69400 Could one of you provide a link to the repository so I can reproduce this issue?

**Comment by meriturva** [comment]
Same here. I think it is not related to repo but to the latest version of vs code

**Comment by PsykoSoldi3r** [comment]
I could not reproduce this issue, on my side it looks fine with the latest version of vscode. Could someone share an repository so I could reproduce the issue

**Comment by mambari** [comment]
I just added you to one of my repo.


Le lun. 15 mars 2021 à 12:20, Joey Kamsteeg ***@***.***> a
écrit :

**Comment by PsykoSoldi3r** [comment]
@mambari Could you please share your git flow config? Running the following commands in the terminal:
```
git config gitflow.branch.master
git config gitflow.branch.develop
git config gitflow.prefix.feature
git config gitflow.prefix.release
git config gitflow.prefix.hotfix
git config gitflow.prefix.support
git config gitflow.prefix.versiontag
```

Probably when these are not set the git flow tab will not be displayed. I would like to make sure of that. Could you please share me those details?

I guess only the version tag could not be set and should not be block displaying the git flow tab in vscode.

**Comment by mambari** [comment]
(JiraDashboard) ∞ 12:44 mehdiambari ➜  JiraDashboard git:(develop) ✗ git config gitflow.branch.master
git config gitflow.branch.develop
git config gitflow.prefix.feature
git config gitflow.prefix.release
git config gitflow.prefix.hotfix
git config gitflow.prefix.support
git config gitflow.prefix.versiontag
main
develop
feature/
release/
hotfix/

(JiraDashboard) ∞ 12:44 mehdiambari ➜  JiraDashboard git:(develop) ✗

**Comment by joeykamsteeg** [comment]
@mambari I have released a new version 0.5.1 with this fix. Could you please confirm it is solved on your side?

I have made git flow config master and develop branches required to be set to show the git flow tab.

**Comment by mambari** [comment]
Yes! Thank you it works.
![image](https://user-images.githubusercontent.com/10997358/111150426-5ef69280-858e-11eb-8c2d-3e0827b92b1a.png)
